Assets/Scripts/PlayerSupervisor.cs

```diff
@@ -88,6 +99,9 @@ void Start()
             paddle = FindObjectOfType<Paddle>();
         paddleOffset = paddle.transform.localPosition;
 
+        if(!trainingBlocksGroup)
+            trainingBlocksGroup = GameObject.Find("TrainingBlocksGroup");
+
```

Comment:
Instead of requiring an empty object to be in the scene, it would be easier just to instantiate a container object here so you don't have to deal with another prefab.

Reply:
thanks for the extra info on Slack. this has been updated.